backend/danswer/llm/chat_llm.py

```diff
@@ -209,7 +211,15 @@ def __init__(
         self._api_base = api_base
         self._api_version = api_version
         self._custom_llm_provider = custom_llm_provider
-        self._max_output_tokens = max_output_tokens
+        self._max_output_tokens = (
```

Comment:
this is not used either, right? If so, can we comment it + add brief comment as to why